operator/pkg/model/ingestion/ingress.go

```diff
@@ -133,6 +133,18 @@ func Ingress(ing networkingv1.Ingress, defaultSecretNamespace, defaultSecretName
 
 	secureListenerMap := make(map[string]model.HTTPListener)
 
+	// Before we check for TLS config, we need to see if the force-https annotation
+	// is set.
+	forceHTTPsannotation := annotations.GetAnnotationForceHTTPSEnabled(&ing)
+	var forceHTTPs bool
```

Comment:
even though it's the default it would be more expressive to write

`forceHTTPs := false`